### PR TITLE
[RHACS] Revised text for internal and external entities

### PR DESCRIPTION
**Pull request by gaurav-nelson**
For https://issues.redhat.com/browse/ROX-24651

Updated the descriptions for internal and external entities.

Cherry-pick in:
- `rhacs-docs-4.4`
- `rhacs-docs-4.5`

---

NOTE:

- This PR is for RHACS docs.
- It doesn't require any special labels or milestones.

### CODE REVIEW

File: modules/network-graph-20-view.adoc [hunk 1]
```diff
@@ -11,8 +11,8 @@ The network graph provides high-level and detailed information about deployments
...
 
-External entities:: These represent entities that are connected outside of your cluster. For more information, see "External entities and connections in the network graph".
-Internal entities:: These represent entities that are connected inside of your cluster and are not considered external. Internal entities are displayed when Sensor is unable to match connections to existing deployments, but the involved IP address is part of the cluster's internal networks. Based on the IP addresses, the system has determined that those flows do not leave the cluster. For more information about the kind of connections that are considered internal, see "Connections involving internal entities".
+Internal entities:: These represent a connection between a deployment and an IP address that belongs to the private address space as defined in link:https://datatracker.ietf.org/doc/html/rfc1918[RFC 1918]. For more information, see <<internal-entities-connections_{context}>>.
```
Comment (gaurav-nelson) [inline modules/network-graph-20-view.adoc:14]: Checking with team if we can use `<<anchor>>` format in modules.

**Comment by tmalove** [comment]
/remove-label peer-review-in-progress
/label peer-review-done

Preview link: https://77531--ocpdocs-pr.netlify.app/openshift-acs/latest/operating/manage-network-policies.html#network-graph-20-view_manage-network-policies

**Pull request merged**